src/contrib/serializers/Akka.Serialization.Hyperion/HyperionSerializer.cs

```diff
@@ -170,7 +170,8 @@ public sealed class HyperionSerializerSettings
         /// <summary>
         /// Default settings used by <see cref="HyperionSerializer"/> when no config has been specified.
         /// </summary>
-        public static readonly HyperionSerializerSettings Default = Create(HyperionSerializer.DefaultConfiguration());
+        public static readonly HyperionSerializerSettings Default = Create(HyperionSerializer.DefaultConfiguration()
+            .GetConfig("akka.actor.serialization-settings.hyperion"));
```

Comment:
This is the actual config `HyperionSerializerSettings` was expecting, not the root config.